spec/openapi.yaml

```diff
@@ -264,6 +278,7 @@ paths:
     get:
```

Comment:
What is the difference between `/installs/{listing_id}` and `/installs/{listing_id}/metrics`? They have different filtering options but return the same schema, so they look like they're just different "flavours" of the same endpoint?

Reply:
The fact that these return the same data was an error on my part, apologies. I'll create an issue to track that change. `/installs/{id}` returns a simple count of installs for a given listing. This call can be more accurate than the general listing count which will be aggregated on a schedule rather than an on-demand count which is a heavy performance call. This can also have filters to represent how many installs were tracked for the given filter set, allowing some drilldown analytics of a given listing.

`/installs/{listing_id}/metrics` returns the overall metrics for a given yearly period and uses previously aggregated data to make the response much faster than on-demand queries.